jcsv.go

```diff
@@ -9,3 +14,62 @@ func CsvToJson(c []byte, hasHeaders bool) ([]byte, error) {
 	// TODO: convert CSV data in `c` into JSON format and return
 	return nil, nil
 }
+
+func main() {
```

Comment:
There shouldn't be a `main()` function in the package `jcsv`. You can add this for testing, but don't commit it. For testing you can write proper `_test.go` files.

Reply:
Sure, I will follow this paradigm from now on.